docs/news/governance.md

```diff
@@ -1,20 +1,21 @@
 # News / Governance
 
 ## **Featured**
-[<h2>Title</h2>](URI_to_news) |
+[<h2>Polkadot's Plan for Governing a Blockchain of Blockchains<h2>](https://www.coindesk.com/polkadots-radical-plan-governing-blockchain-blockchains/) | 

```

Comment:
This also should have the two empty columns in the table to make it properly sized.

Reply:
Also the Polkadot item should be included in the listing below besides being a featured item, as you have done for example with `daos.md`.